src/core/server/http/cookie_session_storage.test.ts

```diff
@@ -103,7 +103,7 @@ interface Storage {
 }
 
 function retrieveSessionCookie(cookies: string) {
-  const sessionCookie = request.cookie(cookies);
+  const sessionCookie = parseCookie(cookies);
```

Comment:
__note:__ unfortunately couldn't test this change as the entire test suite is skipped for the last 5 months :cry: